src/server/create/microservice/controllers/create.js

```diff
@@ -21,24 +21,24 @@ const microserviceCreateController = {
   },
   handler: async (request, h) => {
     const create = request.pre?.create
-    const repositoryName = create.repositoryName
+    const repository = create.repositoryName
```

Comment:
I'd call this `repositoryName`

Reply:
yeh, I would too, but Sonar didn't like it :-)